Tests/test_imagefont.py

```diff
@@ -918,6 +918,46 @@ def test_cbdt_mask(self):
             assert str(e) in ("unimplemented feature", "unknown file format")
             pytest.skip("freetype compiled without libpng or unsupported")
 
+    @skip_unless_feature_version("freetype2", "2.5.1")
+    def test_sbix(self):
+        try:
+            font = ImageFont.truetype(
+                "Tests/fonts/chromacheck-sbix.woff",
+                size=300,
+                layout_engine=self.LAYOUT_ENGINE,
+            )
+
+            im = Image.new("RGB", (400, 400), "white")
+            d = ImageDraw.Draw(im)
+
+            d.text((50, 50), "\uE901", embedded_color=True, font=font)
+
+            with Image.open("Tests/images/chromacheck-sbix.png") as expected:
+                assert_image_similar(im, expected, 1)
+        except IOError as e:
+            assert str(e) in ("unimplemented feature", "unknown file format")
+            pytest.skip("freetype compiled without libpng or unsupported")
```

Comment:
What exactly does 'or unsupported' mean?

Reply:
The tests are a direct copy of the `test_cbdt...` tests, I no longer remember what is the difference. :/ It might be a left-over from before I added the version check, but I'm not sure. Or it might be referring to a version of FreeType compiled with libpng included but CBDT (or in this case SBIX and WOFF) disabled.

I'll change it to `...or without SBIX support`.